### PR TITLE
feat(ui): Export Panel + AsyncComponent

### PR DESCRIPTION
**Pull request by billyvg**
+ fix propType errors for panel header and DateTime

### CODE REVIEW

**Comment by getsentry-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">You should update CHANGES due to the size of this PR</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

File: src/sentry/static/sentry/app/views/settings/components/panelHeader.jsx [hunk 1]
```diff
@@ -18,26 +20,32 @@ const StyledPanelHeader = styled.div`
...
 `;
 
-const StyledPanelHeading = styled(({lightText, ...props}) => <PanelHeading {...props} />)`
+const StyledPanelHeading = styled(({lightText, isFlex, ...props}) => (
+  <PanelHeading {...props} />
+))`
   font-size: inherit;
   text-transform: inherit;
   margin: 0;
+  ${getFlex};
```
Comment (MaxBittker) [inline src/sentry/static/sentry/app/views/settings/components/panelHeader.jsx]: TIL props are passed into these

**Review by MaxBittker — APPROVED** [review]
looks good - although is there a reason heading will support flex and not flex vs just being opinionated on how contents will be positioned, like is it partially for backwards compatibility or is there specific layouts this enables?

**Comment by billyvg** [comment]
Its for backwards compatibility + I wanted to avoid an extra/unnecessary wrapper.

**Comment by MaxBittker** [comment]
that makes sense

File: src/sentry/static/sentry/app/index.js [hunk 2]
```diff
@@ -116,8 +116,10 @@ export default {
 
     Alerts: require('./components/alerts').default,
     AlertActions: require('./actions/alertActions').default,
+    AsyncComponent: require('./components/asyncComponent').default,
```
Comment (dcramer) [inline src/sentry/static/sentry/app/index.js]: heads up i will have another PR waiting on this shortly

**Pull request merged**